plugins/themes/default/styles/pages/catalog.less

```diff
@@ -0,0 +1,47 @@
+/**
+ * @file  plugins/themes/default/styles/pages/catalog.less
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2003-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @brief Styles applying to catalog page
+ * @link templates/frontend/pages/catalog.tpl
+ */
+.page_catalog {
+	.pkp_series_nav_menu{
+
+		border-top: @bg-border;
+		margin: (@triple * 2) -@base;
+		padding: @double;
+		margin-bottom: -(@triple * 2);
+
+		> h2 {
+			font-size: @font-base;
+			font-weight: @bold;
+			display: inline;
+			margin: 0;
+		}
+
+		> ul {
+			display: inline;
+			list-style: none;
+			padding: 0px;
+			> li {
+				display: inline;
+			}
+		}
+
+		@media(min-width: @screen-phone) {
+			margin-left: -@double;
+			margin-right: -@double;
+		}
+
+		@media(min-width: @screen-desktop) {
+			margin-left: -@triple;
+			margin-right: -@triple;
+			padding: @triple;
+		}
+
+	}
+}
```

Comment:
This all looks good. But let's keep the lists in a vertical format on small devices. Here's what that CSS would look like:

```less
		h2 {
			margin: 0;
		}

		ul {
			list-style: none;
			padding: 0px;
		}

		@media(min-width: @screen-phone) {
			margin-left: -@double;
			margin-right: -@double;
			
			h2 {
				font-size: @font-base;
				font-weight: @bold;
				display: inline;
				margin: 0;
			}

			ul {
				display: inline;
				> li {
					display: inline;
				}
			}
		}
```